src/views/mfa-verify/PushForm.js

```diff
@@ -29,6 +31,18 @@ define(['okta'], function (Okta) {
 
     initialize: function () {
       this.enabled = true;
+      this.listenTo(this.options.appState, 'change:isSuccessResponse',
+        function (state, isSuccessResponse) {
+          if (isSuccessResponse && this.settings.get('features.autoPush')) {
+            var autoPush = $('.custom-checkbox [name=autoPush]');
```

Comment:
In general with backbone code, you'll never see an unscoped jquery selector (i.e. just $(selector)). Usually you'll use something like this.$(selector), which is scoped to just the current view.